README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/lalithsuresh/hiero.svg?branch=master)](https://travis-ci.org/lalithsuresh/hiero)
```

Comment:
We could wait until we get a vmware org repo. That seems appropriate. I'll see if my login works for this.